app/services/foreman/plugin.rb

```diff
@@ -243,7 +243,9 @@ def pending_migrations
       pending_migrations = ActiveRecord::Migrator.new(:up, migration_paths).
         pending_migrations
 
-      pending_migrations.size > 0
+      return false unless pending_migrations.size > 0
+      Rails.logger.warn(
+        "Pending migrations: #{pending_migrations.map(&:name).join(', ')}")
     end
```

Comment:
Missing an explicit return value here, it used to return `true` but now will depend on the logger configuration (potentially it could return false if configured for error messages and above.)